packages/expansion-hub/src/ExpansionHub.ts

```diff
@@ -67,13 +67,31 @@ export interface ExpansionHub extends RevHub {
     getI2CReadStatus(i2cChannel: number): Promise<I2CReadStatus>;
 
     // Motor
+    /**
+     * Configure a specific motor
+     * @param motorChannel
+     * @param motorMode 0 for constant power, 1 for constant velocity, 2 for position
+     * @param floatAtZero whether to coast or break when power is set to 0
+     */
     setMotorChannelMode(motorChannel: number, motorMode: number, floatAtZero: boolean): Promise<void>;
```

Comment:
`motorMode` should be typed as an enum for the high-level library

Reply:
Implemented the enum